lib/internal/nsolid_trace.js

```diff
@@ -66,7 +68,14 @@ function extractSpanContextFromHttpHeaders(context, headers) {
   return getApi().trace.setSpanContext(context, spanContext);
 }
 
+const nsolidTracer = new EventEmitter();
+binding.setToggleTracingFn(() => {
+  nsolidTracer.emit('flagsUpdated');
```

Comment:
Why not using a diagnostic_channel for it?

Reply:
We could, I'm not sure though:
- This event is supposed to be used only internally. I'm not sure we want to add this to a diagnostic_channel which indeed would expand our API.
- It's not really hot code at all. I think an EE should be fine.